app/backend/src/couchers/models.py

```diff
@@ -1216,3 +1216,18 @@ def ready_for_retry(self):
 
     def __repr__(self):
         return f"BackgroundJob(id={self.id}, job_type={self.job_type}, state={self.state}, next_attempt_after={self.next_attempt_after}, try_count={self.try_count}, failure_info={self.failure_info})"
+
+
+class UserBlocking(Base):
+    """
+    Table of blocked users
+    """
+
+    __tablename__ = "user_blocking"
+
+    id = Column(BigInteger, primary_key=True)
+
+    blocking_user = Column(ForeignKey("users.id"), nullable=False)
+    blocked_user = Column(ForeignKey("users.id"), nullable=False)
+    blocked = Column(Boolean, nullable=False, default=True)
```

Comment:
Presence of a row in this table indicates that the blocking user has blocked the blocked user. We don't need the bool here (what does it mean to be blocked but have this false?)

Reply:
When unblocking, instead of dropping the row, I change blocked to false (was taught that it's always safer to not delete data)

Reply:
Our table structure is so complicated that to do "soft deletes" in general is a big pain. We've made a conscious decision to maintain only what information is required at the present state of the database.

We will probably need some kind of history as well, but for now we can do with the WAL from postgres if we ever badly need to recover history